ecom_test/src/generic_helpers/generic_order_helpers.py

```diff
@@ -0,0 +1,112 @@
+        self.cur_file_dir = os.path.dirname(os.path.realpath(__file__))  # Gets the directory path of this script
+
+    def create_order(self, additional_args=None, expected_status_code=201):
+        """
+        Creates an order with provided or default payload, and returns the API response.
+
+        Args:
+        Args:
+            additional_args (dict, optional): Additional data to customize the order payload.
+            expected_status_code (int, optional): The expected HTTP status code for the API response.
+
+           Coupon: to add coupon add "coupon_lines" field to "additional_args"
+            Example: "coupon_lines": [{"code": "<MY COUPON CODE>"}]
+        :param product_ids:
+        :param customer_id:
+        :param coupon_code:
+        :param additional_args: Must be a dictionary.
+            Example:
+                order_payload_addition = {
+                                    "line_items": [{"product_id": < some product id >, "quantity": 1}],
+                                    "coupon_lines": [{"code": < some coupon code >}],
+                                    "shipping_lines": [{"method_id": "flat_rate", "method_title": "Flat Rate", "total": "0.00"}]
+                                }
+
+        Returns:
+            dict: The API response after creating the order.
+        """
+        payload_template = os.path.join(self.cur_file_dir, '..', 'data', 'create_order_payload.json')
+
+        # Load the order payload template from a JSON file
+        with open(payload_template) as f:
+            order_payload = json.load(f)
+
+        # Update payload with additional arguments, if provided
+        if additional_args:
+            assert isinstance(additional_args,
+                              dict), f"Parameter 'additional_args' must be a dictionary but found {type(additional_args)}"
+            order_payload.update(additional_args)
+
+        # If no line items are provided, fetch a random product and add it to the order payload
+        if additional_args and "line_items" not in additional_args.keys():
+            rand_product = self.product_dao.get_random_product_from_db(qty=1)
+            rand_product_id = rand_product[0]['ID']
+            order_payload["line_items"] = [{"product_id": rand_product_id, "quantity": 1}]
+
+        # If no shipping lines are provided, add default shipping details to the order payload
+        if additional_args and "shipping_lines" not in additional_args.keys():
+            order_payload["shipping_lines"] = [{"method_id": "flat_rate", "method_title": "Flat Rate", "total": "0.00"}]
+
+        # Make an API call to create the order using the updated payload
+        rs_api = self.order_api_helper.call_create_order(payload=order_payload,
+                                                         expected_status_code=expected_status_code)
+
+        return rs_api
+
+    @staticmethod
+    def verify_order_is_created(order_json, exp_cust_id, exp_products):
```

Comment:
I dont see where you are using this. I think in tcid60 and 61 it would be good to use this function after creating the order.

Reply:
It will be used for "Order" Tests in future